app/src/common/shared/com/igalia/wolvic/ui/OffscreenDisplay.java

```diff
@@ -136,27 +136,6 @@ class OffscreenPresentation extends Presentation {
         public void onCreate(Bundle savedInstanceState) {
 
             super.onCreate(savedInstanceState);
```

Comment:
The change is fine but now `onCreate()` is basically empty (it just calls the superclass) so it could be removed altogether. What do you think.

Reply:
Yeah I guess you're right.